src/main/java/com/GeorgesServer/app/com/GeorgesServer/handler/FilesHandler.java

```diff
@@ -43,6 +50,23 @@ private void setBody() {
         this.body = fileContents;
     }
 
+    private String applyContentType() {
+        if (requestUrl.contains("txt")) {
+            return "text/plain";
+        }
+        if (requestUrl.contains(".png")) {
+            return "image/png";
+        }
+        if (requestUrl.contains(".gif")) {
+            return "image/gif";
+        }
+        if (requestUrl.contains(".jpeg")) {
+            return "image/jpeg";
+        }
+        return "text/http";
```

Comment:
😱 so many if statements! This isn't perfect but it'll do the trick for most cases `URLConnection.guessContentTypeFromName`